.murdock

```diff
@@ -131,6 +136,22 @@ print_worker() {
         echo "-- running on worker ${DWQ_WORKER} thread ${DWQ_WORKER_THREAD}, build number $DWQ_WORKER_BUILDNUM."
 }
 
+test_hash_calc() {
+    local appdir=$1
+    local bindir=$2
+
+    md5sum $appdir/tests/* $bindir/*.elf | md5sum | cut -f1 -d' '
```

Comment:
Why MD5?

Reply:
> Any reason not to use md5?

There are more advanced hashing algorithms, that have a [least likely chance of collision](https://en.wikipedia.org/wiki/SHA-1#Comparison_of_SHA_functions). The question is more, why would you still use MD5 when e.g. SHA1 (while also not very safe anymore in that regard) is as accessible ;-).

Reply:
Least likely chance when under attack. Anyhow, I don't care, will change.

Reply:
>  SHA1

using sha1 now

Reply:
> Could creating a file with the result of `sha1sum $appdir/tests/* $bindir/*.elf` be moved to the riot makefile.

done